### ISSUE TITLE
Add method to request support

### ISSUE THREAD

**Issue by jmgrady** [sillsdev/TheCombine#1233]
Add a method within The Combine to ask for technical support, such as when the response time is very slow, project exports fail, etc.

**Comment by johnthagen** [comment]
Since The Combine is technically open source and anyone can stand up an instance, the specifics should probably be configured at install time rather than hard coded into the application. We'd probably not want support requests from someone else's instance. Though, given how niche of an application the Combine is, that is probably pretty unlikely.

As side information, our user guide has links to all of the SIL online resources, as well as this GitHub repository.

**Comment by jmgrady** [comment]
Good point.  Although it is not the most helpful, a statement like "Contact your system administrator." tells the user that the problem is not from something that they did or that they can fix themselves.  It is generic enough, though, to handle the multiple instances.  Although The Combine is a niche application, we do want to get to the point of providing instructions for groups to create their own instances so this is a very good point that you bring up.

Part of the reason for moving to Kubernetes (#1093) is the expectation that if one of the components crashes, it will be restarted by the control plane.  If that works as hoped, then this may become a moot point.
